cmd/krew/cmd/install.go

```diff
@@ -89,6 +89,9 @@ Remarks:
 			for _, name := range pluginNames {
 				plugin, err := indexscanner.LoadPluginFileFromFS(paths.IndexPluginsPath(), name)
 				if err != nil {
+					if os.IsNotExist(err) {
+						return errors.New(fmt.Sprintf("plugin %q does not exist in the plugin index", name))
```

Comment:
sorry for back and forth but you can use errors.Errorf for this :)

Reply:
No worries, I'll make the change tomorrow